cmd/opentelemetry/app/exporter/elasticsearchexporter/esmodeltranslator/modeltranslator_test.go

```diff
@@ -215,26 +221,16 @@ func TestEmpty(t *testing.T) {
 }
 
 func TestErrorIDs(t *testing.T) {
-	zero64Bytes := make([]byte, 16)
-	binary.LittleEndian.PutUint64(zero64Bytes, 0)
-	binary.LittleEndian.PutUint64(zero64Bytes, 0)
+	var zero64Bytes [16]byte
+	var zero32Bytes [8]byte
 	tests := []struct {
 		spanID  pdata.SpanID
 		traceID pdata.TraceID
 		err     string
 	}{
-		{
-			traceID: pdata.NewTraceID([]byte("invalid-%")),
-			err:     "TraceID does not have 16 bytes",
-		},
```

Comment:
not sure why these tests are removed - give the current code there would still be some kind of error returned from converters.

Reply:
But TraceID is now exactly `[16]byte`. Why would the error be there even if only several first bytes are set?

Reply:
They are silently truncated according to this PR conversations https://github.com/open-telemetry/opentelemetry-collector/pull/2001